data/data/agent/files/usr/local/bin/start-cluster-installation.sh.template

```diff
@@ -7,10 +7,19 @@ wait_for_assisted_service
 
 BASE_URL="{{.ServiceBaseURL}}api/assisted-install/v2"
 
-# Get cluster id
-cluster_id=$(curl -s -S "${BASE_URL}/clusters" | jq -r .[].id)
-# Get infra_env_id
-infra_env_id=$(curl -s -S "${BASE_URL}/infra-envs"| jq -r .[].id)
+cluster_id=""
+infra_env_id=""
+while [[ "${infra_env_id}" = "" || "${cluster_id}" = "" ]]
+do
+    # Get cluster id
+    cluster_id=$(curl -s -S "${BASE_URL}/clusters" | jq -r .[].id)
+    # Get infra_env_id
+    infra_env_id=$(curl -s -S "${BASE_URL}/infra-envs"| jq -r .[].id)
+    if [[ "${infra_env_id}" = "" || "${cluster_id}" = "" ]]; then
+        sleep 2
+    fi	
+done
+echo "Infra env id is $infra_env_id"
```

Comment:
Just a very minor thing. Please print this message on a new line else we get extra periods `.`  from `wait_for_assisted_service`

May 12 14:12:17 master-0.ostest.test.metalkube.org systemd[1]: Starting Service that starts cluster installation...
May 12 14:12:17 master-0.ostest.test.metalkube.org start-cluster-installation.sh[2385]: Waiting for assisted-service to be ready
May 12 14:12:24 master-0.ostest.test.metalkube.org start-cluster-installation.sh[2385]: `.`Infra env id is 8c8e849b-47ad-446a-a592-f33a2c15b276